API.md

```diff
@@ -259,6 +259,19 @@ This page documents the Party service API endpoints. All endpoints return an `HT
 ```
 &mdash; This endpoint will enrol a respondent in a survey and associate with business if not already associated.
 
+
+* `PUT /respondents/edit-account-status`
+
+### Example JSON data for put
+```json
+{
+    "party_id": "438df969-7c9c-4cd4-a89b-ac88cf0bfdf3",
```

Comment:
API docs will need amending to show that we're now including the party ID in the URL and not the JSON.